scripts/install_chrome_on_travis_test.py

```diff
@@ -0,0 +1,87 @@
+# coding: utf-8
+#
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for scripts/install_chrome_on_travis.py."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import subprocess
+
+from core.tests import test_utils
+import python_utils
+from scripts import install_chrome_on_travis
+
+
+class InstallChromeOnTravisTests(test_utils.GenericTestBase):
+
+    def test_main(self):
+        chrome_source_url = 'https://github.com/webnicer/chrome-downloads/raw/master/x64.deb/google-chrome-stable_77.0.3865.75-1_amd64.deb' # pylint:disable=line-too-long
```

Comment:
You don't need the pragma. Do something like this:

```
    chrome_source_url = (
        'https://github.com/webnicer/chrome-downloads/raw/master/x64.deb'
        '/google-chrome-stable_77.0.3865.75-1_amd64.deb')
```

Reply:
Updated